### ISSUE TITLE
All users support on Windows

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#267]
**Reported by Tech.salvager on 23 Oct 2008 17:21:22 UTC**
Yes I moved from xchat to kvirc loving the features and extras kvirc has.  Now I run on my windows computer under LUA (Limited User Accout\Access).  To install apps I usually right click and hit runas and install them using a administrator account, of course I did the same with kvirc installer.  Unfourntaly it gives you no options to install it for all users.

So I'm requesting\asking for a feature request of all user support on windows to be implmented into kvirc.  Its a good app and looks like it has already the correct paths for saving items even under a LUA account.

Basically asking for All User support.
this is in both the most recenet debug  build of 3.4.x and the official build

Tested on windows xp pro sp2 with kvirc 3.4.2 and 3.4.0

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/267

**Comment by DarthGandalf** [comment]
**Comment by Tech.salvager on 23 Oct 2008 17:25:45**
I forgot to include the variable that should/could be used for this
ALLUSERSPROFILE=D:\Documents and Settings\All Users

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 23 Oct 2008 20:13:07**
KVIrc installs its files in a path reachable by default by every user. As far as i understand your problem, it's just that menu entries/desktop icon are installed just for the current user (c:\doc&set\current_user..) and not in the all users dir (c:\doc&set\All users..).
IF this is the problem, win32 packagers can just tune the .nsi (nullsoft installer) script to place these links into the right path. Please correct me if i misunderstood the issue.

**Comment by DarthGandalf** [comment]
**Modified by ctrlaltca on 12 Nov 2008 16:45:53**
Version: 3.4.2 Shiny → 4.0 Insomnia

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 15 Mar 2009 17:49:53**
fixed in r3146

---

Resolution: none → fixed
Status: new → closed
